src/lib/eslint/TypescriptEslintStrategy.ts

```diff
@@ -75,7 +75,7 @@ export class TypescriptEslintStrategy implements EslintStrategy {
 		this.untypedConfig = (await import(pathToUntypedRecommendedConfig)) as ESRuleConfig;
 
 		const pathToTypedRecommendedConfig = require.resolve('@typescript-eslint/eslint-plugin')
-			.replace('index.js', path.join('configs', 'recommended-requiring-type-checking.js'));
+			.replace('index.js', path.join('configs', 'recommended-type-checked.js'));
```

Comment:
Note that the latest @typescript-eslint module has a new name for the recommended typed checking file - so I updated this.